projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/write-linked-cases-field.component.ts

```diff
@@ -39,11 +39,10 @@ export class WriteLinkedCasesFieldComponent extends AbstractFieldWriteComponent
     // Clear validation errors
     this.caseEditDataService.clearFormValidationErrors();
     // Get linked case reasons from ref data
-    this.getLinkedCaseReasons();
     this.linkedCasesService.editMode = false;
-    this.caseEditDataService.caseDetails$.subscribe({
-      next: caseDetails => this.initialiseCaseDetails(caseDetails)
+    this.caseEditDataService.caseDetails$.subscribe({next: caseDetails => { this.initialiseCaseDetails(caseDetails);  }
```

Comment:
You need to unsubscribe from all your subscriptions
Kinda ugly on one line like this